src/Microsoft.Health.Fhir.R5.Core/Microsoft.Health.Fhir.R5.Core.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Microsoft.Health.Fhir.Core</RootNamespace>
+  </PropertyGroup>
+  <PropertyGroup>
+    <DefineConstants>R5</DefineConstants>
+    <NoWarn>$(NoWarn);NU1603</NoWarn>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove="Features\Definition\unsupported-search-parameters.json" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <EmbeddedResource Include="Features\Definition\search-parameters.json" />
+    <EmbeddedResource Include="Features\Definition\compartment.json" />
+    <EmbeddedResource Include="Features\Definition\unsupported-search-parameters.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AngleSharp" Version="0.13.0" />
+    <PackageReference Include="Ensure.That" Version="8.1.2" />
+    <PackageReference Include="FluentValidation" Version="8.6.1" />
+    <PackageReference Include="MediatR" Version="8.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
```

Comment:
Probably not your change, but hadn't noticed this AspNetCore assembly slip into core

Reply:
I'll file a separate bug to fix this, it's there for R4 and Stu3 and removing it causes a couple of unit tests to fail.

Reply:
https://github.com/microsoft/fhir-server/issues/926